src/scenes/gamerulesscreen.ts

```diff
@@ -0,0 +1,102 @@
+    scrollMask: Nullable<Phaser.Display.Masks.GeometryMask>;
+
+    public constructor() {
+        super("GameRulesScreen");
+        this.title = null;
+        this.scrollContainer = null;
+        this.scrollMask = null;
+    }
+
+    public preload(): void {
+        this.load.image("back", "assets/buttons/back.png");
+        this.load.json("rules", "assets/gameRules.json");
+    }
+
+    public create(): void {
+        this.addTitle();
+        this.addScrollableText();
+
+        const backButton: Phaser.GameObjects.Image = this.add.image(
+            ScreenWidth * 0.5,
+            ScreenHeight / 1.25,
+            "back",
+        );
+        interactify(backButton, 0.5, () => this.onBackButton());
+    }
+    private addTitle(): void {
+        this.title = this.add
+            .text(ScreenWidth / 2, ScreenHeight / 8, "Saboteur Rules", {
+                fontFamily: "Verdana",
+                fontSize: 38,
+                color: "#ffffff",
+                stroke: "#000000",
+                strokeThickness: 8,
+                align: "center",
+            } as Phaser.Types.GameObjects.Text.TextStyle)
+            .setOrigin(0.5);
+    }
+
+    private addScrollableText(): void {
+        const scrollAreaHeight: int = ScreenHeight / 2;
+        const scrollAreaWidth: int = ScreenWidth * 0.8;
+        const scrollAreaY: int = ScreenHeight / 4;
+        const rules: string = this.getRules();
+
+        this.scrollContainer = this.add.container(ScreenWidth / 2, scrollAreaY);
+        this.scrollContainer.setSize(scrollAreaWidth, scrollAreaHeight);
+
+        const rulesText = this.add.text(0, 0, rules, {
+            fontFamily: "Verdana",
+            fontSize: 20,
+            color: "#ffffff",
+            wordWrap: { width: scrollAreaWidth - 20 },
+        });
+        rulesText.setOrigin(0.5, 0);
+        this.scrollContainer.add(rulesText);
+
+        const maskShape = this.make.graphics({ x: 0, y: 0, add: false });
+        maskShape.fillStyle(0xffffff);
+        maskShape.fillRect(
+            (ScreenWidth - scrollAreaWidth) / 2,
+            scrollAreaY,
+            scrollAreaWidth,
+            scrollAreaHeight,
+        );
+
+        this.scrollMask = maskShape.createGeometryMask();
+        this.scrollContainer.setMask(this.scrollMask);
+
+        this.input.on(
+            "wheel",
+            (pointer, gameObjects, deltaX, deltaY, deltaZ): void => {
+                if (this.scrollContainer) {
+                    this.scrollContainer.y -= deltaY;
+                    this.scrollContainer.y = Phaser.Math.Clamp(
+                        this.scrollContainer.y,
+                        ScreenHeight / 4 - rulesText.height + scrollAreaHeight,
+                        ScreenHeight / 4,
+                    );
+                }
+            },
+        );
+    }
+
+    private onBackButton(): void {
+        this.scene.stop("GameRuleScreen");
+        this.scene.start("LobbyScreen");
```

Comment:
did you try just to start the lobby? i think it should auto stop the rule screen.